branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -57,6 +57,11 @@ public IBranchLayout slideOut(String branchName) {
     return new BranchLayout(rootEntries.flatMap(rootEntry -> slideOut(rootEntry, branchName)));
   }
 
+  @Override
+  public List<IBranchLayoutEntry> getEntriesOrderedList() {
```

Comment:
I was wondering about refactoring also in `getNext` and `getPrevious` because the code is there. albeit, add a further step to return only the names instead of the layout entries. WDYT?